arbos/l2pricing/model.go

```diff
@@ -87,10 +87,6 @@ func (ps *L2PricingState) UpdatePricingModel(l2BaseFee *big.Int, timePassed uint
 		arbmath.BigFloatMulByUint(rateRatio, uint64(oneInBips-poolWeight)),
 	).Uint64()
 	averageOfRatios := arbmath.Bips(averageOfRatiosRaw)
-	averageOfRatiosUnbounded := averageOfRatios
-	if averageOfRatios > arbmath.PercentToBips(200) {
-		averageOfRatios = arbmath.PercentToBips(200)
-	}
```

Comment:
This should happen conditionally to format version 3, to avoid breaking compatibility with the current devnet